usaspending_api/api_contracts/contracts/v2/search/spending_by_award_count.md

```diff
@@ -61,7 +66,7 @@ This endpoint takes award filters, and returns the number of awards in each awar
     Award IDs surrounded by double quotes (e.g. `"SPE30018FLJFN"`) will perform exact matches as opposed to the default, fuzzier full text matches.  Useful for Award IDs that contain spaces or other word delimiters.
 + `award_amounts` (optional, array[AwardAmounts], fixed-type)
 + `program_numbers`: `10.331` (optional, array[string])
-+ `naics_codes` (optional, NAICSCodeObject)
++ `naics_codes` (optional, NAICSCodesObject)
```

Comment:
The object name was correct

Reply:
👍

Reply:
@tony-sappe also I just added the commit from the other PR which moves the documentation out of the readme, per your first review.

Reply:
`NAICSCodeObject` is used in at least 12 API contracts. If the name needs to change you'll need to update every API contract